gstreamer-send/README.md

```diff
@@ -51,6 +51,10 @@ These pipelines work on Linux, they may have issues on other platforms. We would
 
   `echo $BROWSER_SDP | gstreamer-send -video-src "autovideosrc ! video/x-raw, width=320, height=240 ! videoconvert ! queue"`
 
+* a screen, with computer generated audio.
```

Comment:
I don't think this description matches the actual pipeline as there's no generated audio. You could say something like
> live screen capture
or 
> live desktop capture